src/views/statutory/PositionsList.vue

```diff
@@ -252,6 +271,28 @@ export default {
         }
       })
     },
+    deletePosition (position) {
+      this.$buefy.dialog.confirm({
+        title: 'Removing position',
+        message: 'Are you sure you want to remove the position for <b>' + position.name + '</b>? This action cannot be undone.',
+        confirmText: 'Remove position',
+        type: 'is-danger',
+        hasIcon: false,
+        onConfirm: () => {
```

Comment:
let's move it into a different method, this is what we have all over the project

basically have `askDeletePosition`, which is a confirmation, then `deletePosition` which does the deletion actually